### PR TITLE
Adds different light colors to flamer fire

### PR DESCRIPTION
**Pull request by Spyroshark**
## About The Pull Request
This makes fire have different colored light with an intensity based on how strong the fire is.
It also makes light from flares colored.

Normal fire:
firered

Blue fire
fireblue

Green fire
firegreen

flarelight

## Why It's Good For The Game

It looks nice, and makes good use of dynamic lighting.

## Changelog
:cl:
add: Fire will now make colored light, based on the color of the fire.
/:cl:

### CODE REVIEW

**Comment by Spyroshark** [comment]
I only now thought to also make flares have colored light, and we already have a predefined light color for flares

**Comment by Spyroshark** [comment]
Here's how the flare looks ingame.
flarelight

File: code/modules/projectiles/updated_projectiles/guns/flamer.dm [hunk 1]
```diff
@@ -546,19 +546,26 @@
 	updatehealth()
 
 /obj/flamer_fire/proc/updateicon()
-	if(burnlevel < 15)
-		color = "#c1c1c1" //make it darker to make show its weaker.
+	var/light_color = "red"
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/guns/flamer.dm]:
```suggestion
	var/light_color = "LIGHT_COLOR_LAVA"
```

Comment (Spyroshark) [reply code/modules/projectiles/updated_projectiles/guns/flamer.dm]: oops, forgot to change that.

**Pull request merged**